src/Dynamo/src/HyparDynamo/Hypar.cs

```diff
@@ -2,55 +2,66 @@
 using System.Collections.Generic;
 using System.Linq;
 using Elements.Serialization.glTF;
-using Elements.Serialization.JSON;
 using Hypar.Revit;
 using RevitServices.Persistence;
 
-
 namespace HyparDynamo.Hypar
 {
     public static class Wall
     {
         /// <summary>
-        /// gets the walls
+        /// Convert Revit walls to Elements.WallsByProfiles for use in Hypar models
         /// </summary>
-        /// <param name="wall">The walls to be exported</param>
-        /// <returns name="Hypar.Wall">The Hypar Wall element </param>
-        public static Elements.Wall[] FromRevitWall( this Revit.Elements.Wall RevitWall) 
+        /// <param name="RevitWall">The walls to be exported</param>
+        /// <returns name="WallByProfile">The Hypar walls</param>
+        public static Elements.WallByProfile[] FromRevitWall(this Revit.Elements.Wall RevitWall)
```

Comment:
Can we capitalization consistent here ex: `revitWall` and use the xml to override it?

Reply:
I can't find a way to overide a parameter name.  only class and namespaces names.  
It appears that camelCase is the node input naming standard in dynamo, so we should be good with just fixing this one.

Reply:
this is the last status I could find regarding this feature in Dynamo
https://github.com/DynamoDS/Dynamo/issues/6360